### PR TITLE
Update ms-cloud-germany-transition-add-pre-work.md

### PR DESCRIPTION
**Pull request by Andy-0-Ryan**

### CODE REVIEW

**Comment by JoeDavies-MSFT** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 1]
```diff
@@ -110,18 +110,19 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
 | Notify users of required IMAP4/POP3/SMTP client changes. | Users who have device connections to Microsoft Cloud Deutschland endpoints for client protocols IMAP4, POP3, SMTP are required to manually update their client devices to switch to the [Office 365 worldwide endpoints](https://docs.microsoft.com/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide). | Pre-communicate this dependency to users of these protocols and ensure they either switch to use Outlook mobile or Outlook on the web during this migration. Failure to update client endpoints will result in client connection failures against Microsoft Cloud Deutschland when user mailboxes are migrated. |
 ||||
 
-### Exchange Online Hybrid configuration
+### Exchange Online Hybrid Customers
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md]:
```suggestion
### Exchange Online Hybrid customers
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 2]
```diff
@@ -110,18 +110,19 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
...
...k to customer Active Directory via AAD Connect. The result is a descrepency between the Active Directory representation of the mailbox RemoteRecipientType and RemoteDisplayType values and that in Exchange Online defining the mailbox as shared. <br><br> Customer is responsible to ensure that all Shared mailboxes are properly provisioned using `New-RemoteMailbox -Shared`, `Enable-RemoteMailbox -Shared`, or `Set-RemoteMailbox -Shared`.  See reference on how to [Convert a user's mailbox in a hybrid environment](https://docs.microsoft.com/en-gb/microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox?view=o365-worldwide).| Failing to complete this task before Phase 5 [Exchange Online Migration] may result in NDRs for Shared Mailboxes which convert back to unlicensed mailboxes and loss of shared access for affected mailboxes. [Shared mailboxes are unexpectedly converted to user mailboxes after directory synchronization runs in an Exchange hybrid deployment](https://docs.microsoft.com/en-us/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes) outlines the impact of not addressing this before Exchange Online Migration completes.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md]:
```suggestion
| (Pre-Phase 5) - Preserving Shared Mailbox settings | Some Hybrid customers have converted cloud user mailboxes to be 'shared' mailboxes using Exchange Online commands. This cloud mailbox configuration is written to the mailbox and local Exchange Online directory, however, it is not synced back to the customer's Active Directory via AAD Connect. The result is a discrepancy between the Active Directory representation of the mailbox RemoteRecipientType and RemoteDisplayType values and that in Exchange Online defining the mailbox as shared. <br><br> The customer is responsible to ensure that all Shared mailboxes are properly provisioned using `New-RemoteMailbox -Shared`, `Enable-RemoteMailbox -Shared`, or `Set-RemoteMailbox -Shared`.  See this reference for how to [Convert a user's mailbox in a hybrid environment](https://docs.microsoft.com/microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox?view=o365-worldwide).| Failing to complete this task before Phase 5 [Exchange Online Migration] may result in NDRs for Shared Mailboxes which convert back to unlicensed mailboxes and loss of shared access for affected mailboxes. [Shared mailboxes are unexpectedly converted to user mailboxes after directory synchronization runs in an Exchange hybrid deployment](https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes) outlines the impact of not addressing this before Exchange Online Migration completes.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**